src/openrpc/chains/sui/sui.yaml

```diff
@@ -55,6 +55,111 @@ methods:
+                type: string
+                description: Amount of base asset canceled in the order.
+              client_order_id:
+                type: string
+                description: Client-defined identifier for the order.
+              is_bid:
+                type: boolean
+                description: Indicates whether the order was a bid.
+              order_id:
+                type: string
+                description: Unique ID of the canceled order.
+              original_quantity:
+                type: string
+                description: Original quantity of the order before cancellation.
+              pool_id:
+                type: string
+                description: Pool identifier where the order was placed.
+              price:
+                type: string
+                description: Price level associated with the canceled order.
+              timestamp:
+                type: string
+                description: Timestamp of the event in milliseconds.
+              trader:
+                type: string
+                description: Address of the trader associated with the order.
+          bcsEncoding:
+            type: string
+            description: Encoding format used for the binary canonical serialization (BCS) payload.
+          bcs:
+            type: string
+            description: Base64-encoded BCS data representing the raw event.
+    examples:
+      - request:
+          method: sui_getEvents
+          params:
+            - "5PCRAN3BLaHJsFcVx8MikVrwFyJ53q4YGMqJjhTtvVmF"
+        response:
+          result:
+            - id:
+                txDigest: "5PCRAN3BLaHJsFcVx8MikVrwFyJ53q4YGMqJjhTtvVmF"
+                eventSeq: "0"
+              packageId: "0xcaf6ba059d539a97646d47f0b9ddf843e138d215e2a12ca1f4585d386f7aec3a"
+              transactionModule: "pool"
+              sender: "0xffd4f043057226453aeba59732d41c6093516f54823ebc3a16d17f8a77d2f0ad"
+              type: "0x2c8d603bc51326b8c13cef9dd07031a408a48dddb541963357661df5d3204809::order::OrderCanceled"
+              parsedJson:
+                balance_manager_id: "0x47dcbbc8561fe3d52198336855f0983878152a12524749e054357ac2e3573d58"
+                base_asset_quantity_canceled: "39900000000"
+                client_order_id: "5697276"
+                is_bid: true
+                order_id: "8447391300650109801851172"
+                original_quantity: "39900000000"
+                pool_id: "0x56a1c985c1f1123181d6b881714793689321ba24301b3585eec427436eb1c76d"
+                price: "457933"
+                timestamp: "1752846629592"
+                trader: "0xffd4f043057226453aeba59732d41c6093516f54823ebc3a16d17f8a77d2f0ad"
+              bcsEncoding: "base64"
+              bcs: "R9y7yFYf49UhmDNoVfCYOHgVKhJSR0ngVDV6wuNXPVhWocmFwfESMYHWuIFxR5NokyG6JDAbNYXuxCdDbrHHbSTph///////zfwGAAAAAAD87lYAAAAAAP/U8EMFciZFOuullzLUHGCTUW9Ugj68OhbRf4p30vCtzfwGAAAAAAABAK85SgkAAAAArzlKCQAAANj6zB2YAQAA"
```

Comment:
RPC generation is failing because this isn't following OpenRPC schema. For example the top of this method should look like:
```yaml
  - name: sui_getEvents
    summary: Get Events by Transaction Digest
    description: Returns all events emitted during the execution of a transaction, identified by its digest.
```
See other methods here for examples of how to format it